File: src/my_project_typescript/easy_problems_typescript/from101to150/findTargetIndicesAfterSortingArray.ts
function targetIndices(nums: number[], target: number): number[] {

    nums = nums.sort( (a,b) => a<=b?-1:1)
    let len_nums: number = nums.length
    let answer: Array<number> = []

    for (let i = 0; i < len_nums; i++){
        if (nums[i] === target){
            answer.push(i)
        }
    }

    return answer 
};

console.log(targetIndices([1,2,5,2,3],  2))

console.log(targetIndices([1,2,5,2,3], 3))

console.log(targetIndices([1,2,5,2,3],  5))

console.log(targetIndices([1,2,5,2,3],  4))

File: src/my_project/easy_problems/from201to250/find_target_indices_after_sorting_array.py
from typing import List 

class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        
        nums.sort()

        len_nums: int = len(nums)
        answer: List[int] = list()
        
        for i in range(len_nums):
            if nums[i] == target:
                answer.append(i)
                        
        
        return answer 
    
    
solution = Solution()

print(solution.targetIndices(nums = [1,2,5,2,3], target = 2))

print(solution.targetIndices(nums = [1,2,5,2,3], target = 3))

print(solution.targetIndices(nums = [1,2,5,2,3], target = 5))

print(solution.targetIndices( nums = [1,2,5,2,3], target = 4))
